.devcontainer/launch.sh

```diff
@@ -11,17 +11,38 @@ print_help() {
     echo "the top-level devcontainer in .devcontainer/devcontainer.json will be used."
     echo ""
     echo "Options:"
-    echo "  -c, --cuda       Specify the CUDA version. E.g., 12.2"
-    echo "  -H, --host       Specify the host compiler. E.g., gcc12"
-    echo "  -d, --docker     Launch the development environment in Docker directly without using VSCode."
-    echo "  -h, --help       Display this help message and exit."
+    echo "  -c, --cuda               Specify the CUDA version. E.g., 12.2"
+    echo "  -H, --host               Specify the host compiler. E.g., gcc12"
+    echo "  -d, --docker             Launch the development environment in Docker directly without using VSCode."
+    echo "  --gpus gpu-request       GPU devices to add to the container ('all' to pass all GPUs)."
+    echo "  -e, --env list           Set additional container environment variables."
+    echo "  -v, --volume list        Bind mount a volume."
+    echo "  -h, --help               Display this help message and exit."
+}
+
+_upvar() {
```

Comment:
readability: let's use a more descriptive name and maybe a docstring to make this function's purpose clear.

Reply:
Added a docstring that links to https://fvue.nl/wiki/Bash:_Passing_variables_by_reference. The original wiki used the name `upvar`, and it's also used in other notable projects like [Debian's bash-completion-lib](https://salsa.debian.org/debian/bash-completion/-/blob/ea254e6dfd39fe80a7b314d6e5f742ef625ffe86/bash_completion#L263-319). Do you still want to find a different name for this function?